src/components/SingleArticle.jsx

```diff
@@ -30,6 +31,9 @@ if (isLoading) {
         <p>{article.body}</p>
         <img src={article.article_img_url} />
         <p>Author: {article.author}</p>
+        <Link to={`/articles/${article.article_id}/comments`}>
```

Comment:
Something to think about from a UI/UX perspective - would a user want to navigate to a different page to see the comments for an article? Or do we typically expect to see the comments underneath the article - users usually prefer the path of least difficulty, whatever gets them to what they want the quickest! :')
That's not to say this is wrong, you can absolutely structure your website however you prefer - it is all working great :)!

Reply:
Thanks @nc-saima-khanom ! I agree, I planned to create an 'all comments' page first via the 'view all comments' button and then display the 3 top comments on the article page later, similar UX to Instagram :)